src/main/controllers/RespondentApplicationDetailsController.ts

```diff
@@ -52,9 +52,66 @@ export default class RespondentApplicationDetailsController {
       }
     }
 
+    let claimantResponseDocDownload = undefined;
+
+    if (selectedApplication.value?.respondCollection && selectedApplication.value?.respondCollection.length) {
+      for (let i = selectedApplication.value?.respondCollection.length - 1; i >= 0; i--) {
+        if (
+          selectedApplication.value?.respondCollection[i].value.from === 'Claimant' &&
```

Comment:
Could we put selectedApplication.value.respondCollection[i].value in a variable, as we are reusing it?

Do we need the null check here as it is already done above?

Reply:
Refactored